README.md

```diff
@@ -475,6 +502,16 @@ You can subscribe on the following `$SYS` topics to get client presence:
  - `$SYS/+/disconnect/clients` - will inform about client disconnections.
 The payload will contain the `clientId` of the connected/disconnected client
 
+## Plugins
+
+- [aedes-persistence](https://github.com/moscajs/aedes-persistence): In-memory implementation of an Aedes persistence
+  - [aedes-persistence-mongodb](https://github.com/moscajs/aedes-persistence-mongodb): MongoDB persistence for Aedes
+  - [aedes-persistence-redis](https://github.com/moscajs/aedes-persistence-redis): Redis persistence for Aedes
+  - [aedes-persistence-level](https://github.com/moscajs/aedes-persistence-level): LevelDB persistence for Aedes
+  - [aedes-persistence-nedb](https://github.com/ovhemert/aedes-persistence-nedb#readme): NeDB persistence for Aedes
+- [aedes-logging](https://github.com/moscajs/aedes-logging): Logging module for Aedes, based on Pino
+- [aedes-stats](https://github.com/moscajs/aedes-stats): Stats for Aedes
```

Comment:
We should probably include links to the mqemitter equivalents as well.

Reply:
Now I will add missings mqemitters links. COuld you check also if all the ticks on the table of features are correct? I have a '?' on bridge as I don't understand what it means

Reply:
done

Reply:
readme has '?' on bridge. what does it mean ?

Reply:
what i mean to highlight is - aedes either supports bridge or not. so documentation should represent that accurately.